src/patternfly/components/TreeView/tree-view.scss

```diff
@@ -17,22 +17,21 @@ $pf-c-tree-view--MaxNesting: 10;
   --pf-c-tree-view__node--Color: var(--pf-global--Color--100);
   --pf-c-tree-view__node--m-current--Color: var(--pf-global--link--Color);
   --pf-c-tree-view__node--m-current--FontWeight: var(--pf-global--FontWeight--bold);
-  --pf-c-tree-view__node--m-active--FontWeight: var(--pf-global--FontWeight--bold);
 
   // link hover/focus
   --pf-c-tree-view__node--hover--BackgroundColor: var(--pf-global--BackgroundColor--200);
   --pf-c-tree-view__node--focus--BackgroundColor: var(--pf-global--palette--black-200);
 
   // toggle 90degon
-  --pf-c-tree-view__node-toggle-icon--PaddingRight: var(--pf-global--spacer--sm);
-  --pf-c-tree-view__node-toggle-icon--MinWidth: var(--pf-global--spacer--lg);
-  --pf-c-tree-view__node-toggle-icon--MinHeight: var(--pf-c-tree-view__node-toggle--icon--MinWidth);
+  --pf-c-tree-view__node-toggle--PaddingRight: var(--pf-global--spacer--sm);
+  --pf-c-tree-view__node-toggle--MinWidth: var(--pf-global--spacer--lg);
+  --pf-c-tree-view__node-toggle--MinHeight: var(--pf-c-tree-view__node-toggle--icon--MinWidth);
 
```

Comment:
```suggestion

// Toggle icon
--pf-c-tree-view__node-toggle--Transition: var(--pf-global--Transition);
```

Reply:
Looks like it's `.2s ease-in 0s` in data-list and table, but in nav, it's `var(--pf-c-nav__toggle-icon--Transition)` with default easing.